reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Running filtered test: " + testId.join(", ") +
+			" <a id='qunit-clearFilter' href='" +
+			setUrl({ filter: undefined, module: undefined, testId: undefined }) +
+			"'>" + "Clear filter" + "</a></div>";
+	} else {
```

Comment:
Sure, but you can drop the `else`, rougly like this:
```js
if ( condition ) {
  return stuff;
}
return "";
```
I usually prefer the early return, or "guard clause":
```js
if ( !condition ) {
  return "";
}
/* lots of processing */
return stuff;
```